bdk_keychain/src/keychain_txout_index.rs

```diff
@@ -233,6 +233,15 @@ impl<K: Clone + Ord + Debug> KeychainTxOutIndex<K> {
         (next_derivation_index, new_spk)
     }
 
+    pub fn derive_new_count(&mut self, keychain: &K, count: usize) -> BTreeMap<u32, Script> {
+        let mut out = BTreeMap::new();
+        for _ in 0..count {
+            let (index, script) = self.derive_new(keychain);
+            out.insert(index, script.clone());
+        }
+        out
+    }
```

Comment:
what's wrong with just using `store_up_to` where you pass in the index you want to get to. You can translate any call to `derive_new_count` to `store_up_to` can't you?

Not sure what's going to be done with return value here.